django/db/backends/postgresql/base.py

```diff
@@ -469,6 +484,23 @@ class ServerBindingCursor(CursorMixin, Database.Cursor):
     class Cursor(CursorMixin, Database.ClientCursor):
         pass
 
+    class ServerCursor(
+        CursorMixin, Database.client_cursor.ClientCursorMixin, Database.ServerCursor
+    ):
```

Comment:
References to `ClientCursorMixin` (aka do client-side bindings) and `ServerCursor` (aka use `DECLARE` and friends)
- https://github.com/psycopg/psycopg/blob/ec7a6adfc0b5ffd8d8fb7e247636ab89589c5c0b/psycopg/psycopg/client_cursor.py#L31-L81
- https://github.com/psycopg/psycopg/blob/ec7a6adfc0b5ffd8d8fb7e247636ab89589c5c0b/psycopg/psycopg/server_cursor.py#L212-L349

Reply:
Should we name our cursor `ServerSideCursor` (open to better naming) to not confuse it more than necessarily with the psycopg variant? It is already unfortunate that our `Cursor` class means client-side bindings while the psycopg one uses server-side bindings, but we cannot change that due to the expectation of how our `Cursor` should perform (I guess).

Reply:
I agree that `ServerSideCursor` is a better name. I'll change the name to it.